src/main/java/com/extendedclip/deluxemenus/action/ClickActionTask.java

```diff
@@ -401,6 +413,27 @@ public void run() {
                     }
                 }
 
+                if (isRaw) {

```

Comment:
The "if" statement is not required here? The entire switch statement can also be combined with the one below.